forte/data/data_store.py

```diff
@@ -187,6 +197,76 @@ def _new_tid(self) -> int:
         r"""This function generates a new ``tid`` for an entry."""
         return uuid.uuid4().int
 
+    def _get_type_info(self, type_name: str) -> Dict[str, int]:
+        """
+        Add a new type input_entry_class into ``_type_attributes``.
+        This method will get all attributes of a type and assign index to them.
+        Args:
+            type_name (str): A string or class type representing a class.
+        Returns:
+            attr_dict (dict): The attribute dict of the given type.
+        Raises:
+            RuntimeError: when the type is not provided by ontology file and
+            dynamic import is disabled.
+        """
+        # check if type is in dictionary
+        if type_name in self._type_attributes:
+            return self._type_attributes[type_name]
+        if not self._dynamically_add_type:
+            raise RuntimeError(
+                f"{type_name} is not an existing type in current data store."
+                f"Dynamically add type is disabled."
```

Comment:
As an exception message, it should explain what to do (which boolean needs to be turned on). Further, `ValueError` might be more appropriate.